.github/workflows/build.yml

```diff
@@ -12,7 +12,7 @@ permissions:
   contents: write
 jobs:
   pre-deploy:
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-22.04
```

Comment:
Should this be in a separate PR with a message saying why we're doing this?

Reply:
Yeah, it should really https://github.com/nationalarchives/dr2-ingest/pull/395